bin/proving-service/src/commands/worker.rs

```diff
@@ -92,15 +92,15 @@ impl StartWorker {
         );
 
         // Create a health reporter
-        let (mut health_reporter, health_service) = health_reporter();
+        let (health_reporter, health_service) = health_reporter();
 
         // Mark the service as serving
         health_reporter.set_serving::<ApiServer<RpcListener>>().await;
 
         tonic::transport::Server::builder()
             .accept_http1(true)
-            .add_service(tonic_web::enable(rpc.api_service))
-            .add_service(tonic_web::enable(rpc.status_service))
+            .add_service(rpc.api_service)
+            .add_service(rpc.status_service)
```

Comment:
We no longer need this? I'm assuming we still can make web requests, right?

Also, with this gone, do we still need `tonic-web` dependencies?